deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -279,10 +279,13 @@ class ChatViewController: UITableViewController {
 
     private func startTimer() {
         stopTimer()
-        timer = Timer.scheduledTimer(withTimeInterval: 60, repeats: true) { [weak self] _ in
+        timer = Timer.scheduledTimer(withTimeInterval: 60, repeats: true) { _ in
```

Comment:
sure that `[weak self]` can be removed here? as we're referring self a line below? maybe this is processed differently by swift magic, i do not know much about that, however, if in doubt, i would leave that line using `[weak self] _ in` :)